### ISSUE TITLE
Add operation requirement tests

### ISSUE THREAD

**Issue by MinsukJi-NOAA** [ufs-community/ufs-weather-model#919]
## Description
Make the operation requirement test scripts work for `cpld_bmark_p7` and `cpld_control_p7` tests.

**Comment by MinsukJi-NOAA** [comment]
@DeniseWorthen @junwang-noaa `cpld_bmark_p7` restart test fails to reproduce. I am not entirely sure if I am using all the correct parameters. To compare, I ran a successful restart test for `cpld_control_c96_p7`; please see the comparisons below.

For `cpld_bmark_p7`:
```
[Minsuk.Ji@hfe10 cpld_bmark_p7_rst]$ diff model_configure /scratch1/NCEPDEV/stmp2/Minsuk.Ji/FV3_OPNREQ_TEST/opnReqTest_250759/cpld_bmark_p7_std_base/model_configure 
9c9
< fhrot:                   3
---
> fhrot:                   0
[Minsuk.Ji@hfe10 cpld_bmark_p7_rst]$ diff nems.configure /scratch1/NCEPDEV/stmp2/Minsuk.Ji/FV3_OPNREQ_TEST/opnReqTest_250759/cpld_bmark_p7_std_base/nems.configure 
109c109
<       start_type = continue
---
>       start_type = startup
[Minsuk.Ji@hfe10 cpld_bmark_p7_rst]$ diff input.nml /scratch1/NCEPDEV/stmp2/Minsuk.Ji/FV3_OPNREQ_TEST/opnReqTest_250759/cpld_bmark_p7_std_base/input.nml 
40c40
<   make_nh = .F.
---
>   make_nh = .true.
64c64
<   na_init = 0
---
>   na_init = 1
76c76
<   external_ic = .F.
---
>   external_ic = .true.
79,80c79,80
<   nggps_ic = .F.
<   mountain = .T.
---
>   nggps_ic = .true.
>   mountain = .false.
95c95
<   warm_start = .T.
---
>   warm_start = .false.
172c172
<   nstf_name    = 2,0,0,0,0
---
>   nstf_name    = 2,1,0,0,0

```
For `cpld_control_c96_p7`:
```
[Minsuk.Ji@hfe12 cpld_control_c96_p7_rst]$ diff model_configure /scratch1/NCEPDEV/stmp2/Minsuk.Ji/FV3_OPNREQ_TEST/opnReqTest_199727/cpld_control_c96_p7_std_base/model_configure 
9c9
< fhrot:                   12
---
> fhrot:                   0
[Minsuk.Ji@hfe12 cpld_control_c96_p7_rst]$ diff nems.configure /scratch1/NCEPDEV/stmp2/Minsuk.Ji/FV3_OPNREQ_TEST/opnReqTest_199727/cpld_control_c96_p7_std_base/nems.configure 
95c95
<       start_type = continue
---
>       start_type = startup
[Minsuk.Ji@hfe12 cpld_control_c96_p7_rst]$ diff input.nml /scratch1/NCEPDEV/stmp2/Minsuk.Ji/FV3_OPNREQ_TEST/opnReqTest_199727/cpld_control_c96_p7_std_base/input.nml 
40c40
<   make_nh = .F.
---
>   make_nh = .true.
64c64
<   na_init = 0
---
>   na_init = 1
76c76
<   external_ic = .F.
---
>   external_ic = .true.
79,80c79,80
<   nggps_ic = .F.
<   mountain = .T.
---
>   nggps_ic = .true.
>   mountain = .false.
95c95
<   warm_start = .T.
---
>   warm_start = .false.
172c172
<   nstf_name    = 2,0,0,0,0
---
>   nstf_name    = 2,1,0,0,0
376c376
<   input_filename = 'r'
---
>   input_filename = 'n'
```

**Comment by JessicaMeixner-NOAA** [comment]
@MinsukJi-NOAA currently if you are running with waves (like in cpld_bmark_p7), we expect the restart test to fail to reproduce.

**Comment by MinsukJi-NOAA** [comment]
> @MinsukJi-NOAA currently if you are running with waves (like in cpld_bmark_p7), we expect the restart test to fail to reproduce.

@JessicaMeixner-NOAA Thanks for the clarification.

**Comment by JessicaMeixner-NOAA** [comment]
FYI - same for debug right now with waves.

**Comment by MinsukJi-NOAA** [comment]
#928 merged via #931
